### PR TITLE
Code points for partial file format

### PR DESCRIPTION
**Pull request by jeanlf**
- modified app.js to add new page for checksum registry
- added checksum registry js/vue/csv
- updated brands csv
- updated box type csv

### CODE REVIEW

**Comment by edrthomas** [comment]
Hi @jeanlf , thanks for the PR! Since your adding a page, wouldn't you also like to add an entry in the menu of the page, i.e. in the index.html ?

**Comment by jeanlf** [comment]
oops missed it, now done.

File: CSV/boxes.csv [hunk 1]
```diff
@@ -184,3 +184,18 @@ vmhd,"video media header, overall information (video track only)",ISO
...
+frpa,Front Part,ISO-Partial
+pdat,Partial Data,ISO-Partial
+pfhd,Partial File Header,ISO-Partial
+pfil,Partial File,ISO-Partial
+ploc,Partial Segment Location,ISO-Partial
+pseg,Partial Segment,ISO-Partial
+pshd,Partial Segment Header,ISO-Partial
+ptle,Partial Top Level Entry,ISO-Partial
+surl,Source URL,ISO-Partial
+
```
Comment (dwsinger) [inline CSV/boxes.csv]: blank lines?

File: CSV/brands.csv [hunk 2]
```diff
@@ -148,3 +148,6 @@ ssss,Subsegment Index Segment used to index MPEG-2 TS based Media Segments.,DASH
 uvvu,"UltraViolet file brand – conforming to the DECE Common File Format spec, Annex E.",DECE
 XAVC,XAVC File Format,Sony
 yt4$20,Google specification for use by YouTube apps,Youtube
+paff,"Generic Partial File",ISO-Partial
+pmff,"Mixed Partial File",ISO-Partial
+
```
Comment (dwsinger) [inline CSV/brands.csv]: blank line?

**Pull request merged**